test/saithrift/src/switch_sai_rpc_server.cpp

```diff
@@ -1691,10 +1691,10 @@ class switch_sai_rpcHandler : virtual public switch_sai_rpcIf {
   }
 
   void sai_thrift_get_port_stats(
-          std::vector<uint64_t> & thrift_counters,
+          std::vector<int64_t> & thrift_counters,
```

Comment:
Yes. We can try. But why? I don't see thrift-0.9.3 supports unsigned integers.
https://github.com/apache/thrift/blob/master/CHANGES